dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/TestCaseFixedOmegaRef.mo

```diff
@@ -0,0 +1,98 @@
+within Dynawo.Examples.Nordic;
+
```

Comment:
I have the following error when running the case : ##IDA## linear solver failed try once again = 1.0025607440004
@bongrainmat same for you ?

Reply:
For more information: The testcase runs successfully on my windows machine with OpenModelica 1.18.1/OMSimulator v2.1.1.post80-g1bf17f4-mingw and the old front-end and the version 3.2.3 of the modelica package, using euler or IDA, with the solver settings in the annotations (interval 0.01, tolerance 0.005)

Reply:
On my machine, with OMCompiler v1.17.0 and OMSimulator v2.0.0.post284-gc8ec782-linux, the simulation of TestCaseFixedOmegaRef.mo stops as scheduled at t = 180 s.

Reply:
If I put the terminals before the components in TransformerWithControl.mo (as per the coding style), the simulation stops at t = 167.249 s with the following message :

The initialization finished successfully without homotopy method.
Dynawo.Examples.Nordic.TestCase_16dae.c:29933: Invalid root: (-0.0464225)^(6.0257)
##IDA## -8 error occurred at time = 167.42861443606
model terminate | Integrator failed. | Simulation terminated at time 167.429
### STATISTICS ###
Simulation process failed. Exited with code 247.